FFMpegCore/FFProbe/MediaAnalysis.cs

```diff
@@ -37,6 +48,8 @@ private MediaFormat ParseFormat(Format analysisFormat)
 
         public MediaFormat Format { get; }
 
+        public List<ChapterData> Chapters { get; }
```

Comment:
Will this be null or empty list when the file analysed does not have any chapters?